themes/bootstrap3/templates/Recommend/MapSelection.phtml

```diff
@@ -24,10 +24,6 @@
     $searchParams = $this->recommend->getSearchParams();
     $coordinates = $this->recommend->getSelectedCoordinates();
 
-    if ($coordinates == null) {
```

Comment:
How confident are you that it's safe to remove this logic? I agree that the `$addSearchOption` Javascript below seems to have no effect since I can find no references in the code to a `search-query-options` class, but the `$showSelection` and `$coordinates` values are being passed to the `loadMapSelection` function, and presumably have some effect.

I'm not very familiar with the geo-code, so I want to be sure we don't accidentally break something here. Perhaps @lmgonzales can shed some light if time permits.

Reply:
I tried to experiment a bit with it and even at start of geographic searching, `getSelectedCoordinates` does not return `null` but default coordinates. And `showSelection` parameter is not consumed in `loadMapSelection` so I removed it.

Reply:
Yes, thank you, that makes sense. It appears that there may be some code paths where `getSelectedCoordinates` might be able to return a null value, so out of an abundance of caution, I included `getDefaultCoordinates` as a `??` fallback. It's possible that's not necessary, but it shouldn't hurt anything, and it still dramatically simplifies the code. Here's the final version I pushed to master: https://github.com/vufind-org/vufind/commit/051aae490ff21760b6deb55d2fd43c343939f05c